### PR TITLE
Feat/profile view

### PR DESCRIPTION
**Pull request by Branyoe**
## What
- Added the view that shows the profile information.
- added modal that edits profile information.

## Screenshots
image
image

### CODE REVIEW

File: app/assets/stylesheets/app.scss [hunk 1]
```diff
@@ -6,6 +6,8 @@
   "./sass/posts",
   "./sass/registrations",
   "./sass/new_post",
+  "./sass/edit_view",
+  "./sass/profile_view";
```
Comment (sjlwa) [inline app/assets/stylesheets/app.scss:10]: replace ; to ,

File: app/assets/stylesheets/sass/abstracts/colors.scss [hunk 2]
```diff
@@ -7,4 +7,5 @@ $colorBlue: #28f;
 $colorGray: #888;
 $colorBorder: #2e3139;
 $colorDarkMint: #23484B;
+$colorEmptyBg: #333639
```
Comment (sjlwa) [inline app/assets/stylesheets/sass/abstracts/colors.scss:10]: ; missing

File: app/assets/stylesheets/sass/edit_view.scss
Comment (sjlwa) [inline app/assets/stylesheets/sass/edit_view.scss:1]: sort every attribute alphabetically

File: app/assets/stylesheets/sass/profile_view.scss
Comment (sjlwa) [inline app/assets/stylesheets/sass/profile_view.scss:1]: sort every attribute alphabetically

File: app/assets/stylesheets/app.scss [hunk 3]
```diff
@@ -44,10 +46,6 @@
   width: 3px;
 }
 
-// *::-webkit-scrollbar-track{
-//   -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
-// }
-
```
Comment (sjlwa) [inline app/assets/stylesheets/app.scss:50]: unused

File: app/assets/stylesheets/sass/profile_view.scss [hunk 4]
```diff
@@ -0,0 +1,178 @@
+.embedded{
+  background-color: rgba(89, 89, 92, 0.5);
+  // display: none;
```
Comment (sjlwa) [inline app/assets/stylesheets/sass/profile_view.scss:3]: unused

**Review by sjlwa — CHANGES_REQUESTED** [review]
💯

**Review by sjlwa — APPROVED** [review]
👍🏽

**Review by JoseGabino12 — APPROVED** [review]
LGTM

**Pull request merged**